scss/tech.scss

```diff
@@ -1,31 +1,28 @@
+.tech-container {
+  display: grid;
+  grid-template-columns: repeat(3, 200px);
```

Comment:
I'd change this to `grid-template-columns: repeat(3, auto);` since you  already define the 200px in the tech.